mypyc/genops.py

```diff
@@ -2678,10 +2696,13 @@ def get_final_ref(self, expr: MemberExpr) -> Optional[Tuple[str, Var, bool]]:
         if isinstance(expr.expr, RefExpr) and isinstance(expr.expr.node, TypeInfo):
             # a class attribute
             sym = expr.expr.node.get(expr.name)
-            if sym and isinstance(sym.node, Var) and sym.node.is_final:
-                final_var = sym.node
-                fullname = '{}.{}'.format(sym.node.info.fullname(), final_var.name())
-                native = expr.expr.node.module_name in self.modules
+            if sym and isinstance(sym.node, Var):
+                # Enum attribute are treated as final since they are added to the global cache
+                is_final = sym.node.is_final or expr.expr.node.bases[0].type.name() == 'Enum'
```

Comment:
similar with `fullname`. We should update the enum code in mypy to make the attributes final.